README.md

```diff
@@ -780,6 +788,17 @@ If you need the actual *build time* then simply use the a filtered properties fi
 git.build.time=${git.build.time}
 ```
 
+Note on the git.build.number module
```

Comment:
It's not a module though?

Reply:
K, I will change that. Would "Note on git.build.number variables" work for you?